web/controllers/user_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule MovieWagerApi.UserController do
+  use MovieWagerApi.Web, :controller
+
+  alias MovieWagerApi.{Repo, User, UserSerializer}
+
+  def show(conn, %{"id" => identifier}) do
```

Comment:
zombie records, haha, I love that. Nice, I didn't realize you had found a good way to handle these zombie records on the Ember side... would you share that code with me when it exists? I'd love to know how you fix this issue.